Backend/utils/context.txt

```diff
@@ -0,0 +1,77 @@
+BeatBridge Application Context:
```

Comment:
So is this page needed to explain to llama what the different pages on the site do?

Reply:
Yes to give information about the app